### DISCUSSION TITLE
how to get driver pid

### DISCUSSION THREAD

**Discussion by gouravkumar99** [kaliiiiiiiiii/Selenium-Driverless#90] [Q&A] [answered]
`driver.service.process.pid` is not working

Can you tell me how to get PID of the driver?

**Comment by ttlns** [comment]
@gouravkumar99 please don't open arbitary issues.
The pid can be found in smth similar like `driver._proc.pid` I think

**Reply by gouravkumar99** [reply]
@kaliiiiiiiiii
`driver._proc.pid ` it's not working but `driver.service.process.pid` it's working with other driver

**Reply by ttlns** [reply, marked as answer]
`driver._process.pid` it is in fact

**Reply by gouravkumar99** [reply]
thanks it's working
